web/templates/reference.html

```diff
@@ -51,13 +51,13 @@ <h5 class="text-center">{{ lang_friendlyname }}'s Implementation</h5>
                 </div>
                 <div class="card">
                     <div class="card-body">
-                        <a href="https://github.com/codethesaurus/codethesaur.us/blob/master/web/thesauruses/{{ lang }}/{{ concept_id }}.json">
-                            Want to add or correct information for {{ lang_friendlyname }}?
+                        <a href="https://github.com/codethesaurus/codethesaur.us/blob/master/web/thesauruses/{{ lang|linebreaksbr }}/{{ concept_id|linebreaksbr }}.json">
```

Comment:
Same for URLs here, they shouldn't be broken apart (not that they should ever be on multiple lines anyway).